client/components/table/style.scss

```diff
@@ -58,7 +58,7 @@
 	text-align: left;
 
 	a {
-		&:not(.is-inline) {
+		&:not(.is-not-unique) {
```

Comment:
What does this class mean?

Reply:
I was trying to use a semantic class name, but I just remembered a pseudo-selector exists for that.